docs/dev/system_keyspace.md

```diff
@@ -198,18 +198,22 @@ Holds information about all tablets in the cluster.
 Schema:
 ~~~
 CREATE TABLE system.tablets (
-    keyspace_name text,
     table_id uuid,
     last_token bigint,
+    keyspace_name text static,
     new_replicas frozen<list<frozen<tuple<uuid, int>>>>,
+    repair_scheduler_config text static,
+    repair_time timestamp,
     replicas frozen<list<frozen<tuple<uuid, int>>>>,
+    resize_seq_number bigint static,
+    resize_type text static,
+    session uuid,
     stage text,
-    transition text,
     table_name text static,
     tablet_count int static,
-    resize_type text static,
-    resize_seq_number bigint static,
-    PRIMARY KEY ((keyspace_name, table_id), last_token)
+    tablet_task_info text,
+    transition text,
```

Comment:
There are some unrelated changes, which reflect earlier changes to the schema.

Reply:
So they should be in a separate commit.

Reply:
I realized the doc is out of date for the schema and I used the latest describe schema. I will drop the other changes.

Reply:
But don't drop them from the PR, the other changes are good, they should just be in a separate commit.

Reply:
Let's update the schema in a separate PR. This PR will add what it really adds.